tests/test_backup.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import sys
+import typing
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from scripts import backup
+from fusillade.clouddirectory import cleanup_directory, cleanup_schema
+from tests.common import new_test_directory
+
+
+class Test_Backup(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.directory, cls.schema_arn = new_test_directory()
+
+    @classmethod
+    def tearDownClass(cls):
+        cleanup_directory(cls.directory._dir_arn)
+        cleanup_schema(cls.schema_arn)
+
+    def tearDown(self):
+        self.directory.clear()
+
+    def test_default(self):
+        """
+        Test the backup file is correct after a basic fusillade directory is initialized
+        """
+        contents = backup.backup()
+        # check that it is jsonifyable
+        json.dumps(contents)
```

Comment:
`backup.backup()` would have raised otherwise. This can be omitted.

Reply:
removed